### PR TITLE
Sof 123 receive pod telem

### PR DESCRIPTION
**Pull request by andrewnash**
Should be ready for PDS main integration @mmb186 @spramij

### CODE REVIEW

**Comment by andrewnash** [comment]
This PR is blocking me from beginning heartbeat can I get a review @mmb186 or @lwaghorn

**Comment by spramij** [comment]
A little late to the party but seems perfect to me !

**Review by spramij — APPROVED** [review]

**Review by lwaghorn — APPROVED** [review]

File: OnLand/PDS/Paradigm/pod.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    def __str__(self):
+        return "%s:%d" % self.addrport
+
+
+def main():
+    pod = Pod('127.0.0.1', 6000)
+    pod.connect()
+    
+    while pod.is_connected():
+        data = pod.recv(timedelta(seconds=0.01))
```
Comment (mmb186) [inline OnLand/PDS/Paradigm/pod.py:138]: Can we make timeout a constant.

File: OnLand/PDS/Paradigm/pod.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+    def is_connected(self):
+        return self.sock is not None and self.sock.fileno() >= 0
+
+    def __str__(self):
+        return "%s:%d" % self.addrport
+
+
+def main():
+    pod = Pod('127.0.0.1', 6000)
```
Comment (mmb186) [inline OnLand/PDS/Paradigm/pod.py:134]: Let's make address and port constants.

**Comment by spramij** [comment]
Going further on that, I believe PDS_MAIN branch had started getting environment variables presets, could make use of that. If unsure, could merge and cleanup afterwards

**Comment by andrewnash** [comment]
Added a comment for clarification, the main() is just for testing purposes/ for reference of any other devs of how to implement. As mentioned by @spramij the PDS class will take any env variables as use them accordingly already.

**Review by mmb186 — APPROVED** [review]

**Pull request merged**